uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -7,6 +7,7 @@ define([
   'common/moment',
   'common/services/api/option-group',
   'common/services/hr-settings',
+  'common/services/session',
```

Comment:
Example: 
1. Order of injection for both requirejs and angular
2. Function Declarations to Hide Implementation Details
etc